packages/glab.portable/glab.portable.nuspec

```
@@ -6,14 +6,14 @@
     <version>1.24.1</version>
     <packageSourceUrl>https://github.com/corbob/ChocoPackages/tree/master/packages/glab.portable</packageSourceUrl>
     <owners>corbob</owners>
-    <title>glab (Portable)</title>
+    <title>GLab (Portable)</title>

```

```suggestion
    <title>GitLab CLI (Portable)</title>
```
Let's go with a title that matches what GitLab has labeled it officially: https://gitlab.com/gitlab-org/cli/-/issues/844#note_1201862858